packages/plugin-ext/src/main/node/paths/plugin-paths-service.ts

```diff
@@ -17,13 +17,17 @@
 import { injectable, inject } from 'inversify';
 import { FileSystem, FileStat } from '@theia/filesystem/lib/common';
 import * as path from 'path';
+import {  readdir, remove} from 'fs-extra';
 import * as crypto from 'crypto';
 import URI from '@theia/core/lib/common/uri';
 import { isWindows } from '@theia/core';
 import { PluginPaths } from './const';
 import { PluginPathsService } from '../../common/plugin-paths-protocol';
 import { THEIA_EXT, VSCODE_EXT, getTemporaryWorkspaceFileUri } from '@theia/workspace/lib/common';
 
+const MAX_LOG_SESSIONS_TO_KEEP = 10;
+const SESSION_TIMESTAMP_PATTERN = /\d\d\d\d\d\d\d\dT\d\d\d\d\d\d/;
```

Comment:
```ts
const SESSION_TIMESTAMP_PATTERN = /\d{8}T\d{6}/;
```
Easier to read?

Reply:
Yes. I will fix this later

Reply:
changed to: `/^\d{8}T\d{6}$/` so it would only match **exact* length matches.
`